### PR TITLE
{bio}[iccifort/2019.5.281] Stacks v2.53

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3531.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/dc646250198bfe5c91c1c60eb57b6d64 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3134.skitty.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/93bfe52870fa068c7d8161ab9f964e08 for a full test report.

File: easybuild/easyconfigs/s/Stacks/Stacks-2.53-iccifort-2019.5.281.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+homepage = 'https://catchenlab.life.illinois.edu/stacks'
+description = """Stacks is a software pipeline for building loci from short-read sequences, such as those generated on
+ the Illumina platform. Stacks was developed to work with restriction enzyme-based data, such as RAD-seq,
+ for the purpose of building genetic maps and conducting population genomics and phylogeography.
+"""
+
+toolchain = {'name': 'iccifort', 'version': '2019.5.281'}
+
+source_urls = ['http://catchenlab.life.illinois.edu/stacks/source/']
```
Comment (casparvl) [inline easybuild/easyconfigs/s/Stacks/Stacks-2.53-iccifort-2019.5.281.eb:14]: Doesn't this also have an https equivalent...?

**Comment by casparvl** [comment]
@boegelbot please test @generoso EB_ARGS=--force-download

File: easybuild/easyconfigs/s/Stacks/Stacks-2.53-iccifort-2019.5.281.eb (hunk 1 above)
Comment (casparvl) [reply easybuild/easyconfigs/s/Stacks/Stacks-2.53-iccifort-2019.5.281.eb]: Ok, changed it to https myself, and asked boegelbot to rebuild with force-download. Seems to work :)

**Review by casparvl — APPROVED** [review]
LGTM! Also, no unexpected differences seen with `--review-pr`

**Comment by casparvl** [comment]
Going in, thanks @boegel!

**Pull request merged**